### PR TITLE
Limit content width - Take 2

### PR DESCRIPTION
**Pull request by pradyunsg**
From #11.

Some extra styling to make things look nicer.

Before:

screen shot 2017-12-22 at 1 16 25 am

After:

screen shot 2017-12-22 at 1 14 45 am

### CODE REVIEW

File: pypa_theme/static/sidebar.js [hunk 1]
```diff
@@ -71,7 +72,7 @@ $(function() {
     bodywrapper.css('margin-left', bw_margin_collapsed);
     sidebarbutton.css({
         'margin-left': '0',
-        'height': bodywrapper.height(),
+        'height': documentwrapper.height(),
```
Comment (pradyunsg) [inline pypa_theme/static/sidebar.js]: The reason for making this change is: If there's an element with margin at an extreme, the sidebar didn't take 100% height of the white portions on load.

This fixes that issue.

File: pypa_theme/static/sidebar.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
+/*
```
Comment (jonparrott) [inline pypa_theme/static/sidebar.js]: Why did we end up with a totally new file here for this? Seems unrelated to the body size change?

**Comment by pradyunsg** [comment]
@jonparrot there's an inline comment...

> The reason for making this change is: If there's an element with margin at an extreme, the sidebar didn't take 100% height of the white portions on load.

> This fixes that issue.

**Comment by jonparrott** [comment]
Gotcha, so you're essentially overriding the sidebar.js from the upstream theme?

**Comment by pradyunsg** [comment]
Yep.

**Comment by pradyunsg** [comment]
Sorry that it wasn't clear initially. Something had come up when I made the
PR so I made it in a hurry.

**Comment by jonparrott** [comment]
No worries - just a question: is it possible for us to make the change to that script upstream and have it work for both variants?

**Comment by pradyunsg** [comment]
I didn't wanna fiddle with upstream.

A quick look suggests this exact change will add 1em of height to the lower portion of the sidebar, for example when there's a list involved.

It'll sort of be more consistent across various pages -- currently an element at an extreme that has a margin can cause `bodywrapper` to be shorter than `documentwrapper`. But it's a change no one would notice on the site.

**Comment by pradyunsg** [comment]
In a word: Yes.

**Comment by jonparrott** [comment]
Let's do it then. Python proper still isn't using the upstream theme (I need to pick that thread back up)

**Comment by pradyunsg** [comment]
Cool. I'll drop the change here and put it in there.

**Comment by pradyunsg** [comment]
@jonparrott I've removed sidebar.js and made python/python-docs-theme#9.

**Comment by jonparrott** [comment]
@pradyunsg great, we can merge this. :)

**Comment by pradyunsg** [comment]
Now to trigger a doc rebuild of pip, PyPUG and the rest. :P

**Comment by pradyunsg** [comment]
@jonparrott sphinx 1.7 has an upstream fix for this. I'm tempted to revert the changes here now and just move to the newer version of sphinx.

**Comment by pradyunsg** [comment]
Your thoughts?

**Comment by jonparrott** [comment]
@pradyunsg I'm totally cool with that.
